codec-http/src/main/resources/META-INF/native-image/io.netty/codec-http/native-image.properties

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 The Netty Project
```

Comment:
Probably better removing this header for the resources included in the JAR? (less bloat)

Reply:
Removing this will fail with the checkstyle rules, how should I proceed? I mean the property file will be included in the final artifact published to maven central, so perhaps it should include the header

Reply:
I would leave it, jars are anyway compressed.

Reply:
Yes. Let's leave it as it is.